src/Base/MultiFab.cpp

```diff
@@ -59,16 +59,23 @@ void init_MultiFab(py::module &m) {
         //     [](MFIter & mfi) -> MFIter & {
         //        return mfi;
         //     },
-        //     py::return_value_policy::reference
+        //     py::return_value_policy::reference_internal
         //)
         .def("__next__",
              [](MFIter & mfi) -> MFIter & {
-                ++mfi;
+                static bool first_or_done = true;
```

Comment:
Can you create multiple iterators?

Reply:
I was thinking about this as well, I should make this a state per object